br-webui/views/routing.liquid

```diff
@@ -1,7 +1,7 @@
 {% include ../_includes/header.liquid %}
 
 <script>
-	var socket = io();
+	//var socket = io();
```

Comment:
Please, if you need to remove code you can just delete it, that's why git exists :)

Reply:
Yeah. These were the changes that Jacob made so I didn't do anything with that. I can remove those and push again. :)